WordPress/src/test/java/org/wordpress/android/ui/mysite/cards/quickstart/QuickStartRepositoryTest.kt

```diff
@@ -60,8 +61,17 @@ class QuickStartRepositoryTest : BaseUnitTest() {
     private lateinit var quickStartRepository: QuickStartRepository
     private lateinit var snackbars: MutableList<SnackbarMessageHolder>
     private lateinit var quickStartPrompts: MutableList<QuickStartMySitePrompts>
+    private lateinit var quickStartSiteMenuStep: MutableList<QuickStartSiteMenuStep?>
     private val siteLocalId = 1
 
+    private val siteMenuTasks = listOf(
```

Comment:
❓ for the future: How would this work if we show the qs card on the site_menu and not the dashboard?

Reply:
Actually, `isSiteMenuStepRequiredForTask` checks two conditions:

`quickStartTaskOrigin == MySiteTabType.DASHBOARD && task.showInSiteMenu()`

It doesn't matter which tab we show qs card, what matters is the quick start task origin which gets set on tab change.  Even if the qs card is displayed on the "site menu" tab, if we switch to the "dashboard" tab when the qs prompt is displayed, then we display the "site menu step". Hope it clarifies. 🤞